pkgs/applications/audio/magnetophonDSP/LazyLimiter/default.nix

```diff
@@ -10,7 +11,7 @@ stdenv.mkDerivation rec {
     sha256 = "10xdydwmsnkx8hzsm74pa546yahp29wifydbc48yywv3sfj5anm7";
   };
 
-  buildInputs = [ faust2jaqt faust2lv2 ];
+  buildInputs = [ faust2jaqt faust2lv2 lv2 ];
```

Comment:
Right, this should be fixed by correcting the symlinks in `lv2.dev` to point to `lv2.out`.